ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/component/MessageBubble.kt

```diff
@@ -0,0 +1,136 @@
+package co.electriccoin.zcash.ui.design.component
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.sizeIn
+import androidx.compose.foundation.shape.GenericShape
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import co.electriccoin.zcash.ui.design.theme.ZcashTheme
+
+private const val ARROW_PADDING = 16
+private const val ARROW_WIDTH = 24
+private const val ARROW_HEIGHT = 8
+private const val COMPONENT_MIN_WIDTH = ARROW_WIDTH * 3
+
+@Preview(showBackground = true)
+@Composable
+private fun BubbleWithTextPreview() {
+    ZcashTheme {
+        BubbleMessage(backgroundColor = ZcashTheme.colors.dividerColor) {
+            Text(
+                text = "TextTextTextText",
+                fontSize = 16.sp,
+                modifier = Modifier.padding(ZcashTheme.dimens.spacingDefault)
+            )
+        }
+    }
+}
+
+@Preview("Small content and left arrow preview", showBackground = true)
+@Composable
+private fun BubbleMinSizePreview() {
+    ZcashTheme {
+        BubbleMessage(arrowAlignment = BubbleArrowAlignment.BottomLeft) {
+            Text(
+                text = "T",
+                textAlign = TextAlign.Center,
+                fontSize = 16.sp,
+                modifier = Modifier.padding(ZcashTheme.dimens.spacingDefault)
+            )
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+private fun BubbleWithTextFieldPreview() {
+    ZcashTheme {
+        Box(modifier = Modifier.fillMaxWidth()) {
+            BubbleMessage {
+                @OptIn(ExperimentalFoundationApi::class)
+                FormTextField(
+                    value = "FormTextField",
+                    onValueChange = {},
+                    modifier = Modifier.padding(ZcashTheme.dimens.spacingDefault),
+                    withBorder = false
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun BubbleMessage(
```

Comment:
Just a very minor comment. I noticed we use reverted words in the file and this component name. Should we unify them?